### ISSUE TITLE
Terminal windows fall down with multiple monitors in El Capitan

### ISSUE THREAD

**Issue by quinncomendant** [eczarny/spectacle#536]
I have a new computer with OS X 10.11. I notice that Spectacle 1.0.2 does not handle Terminal windows correctly when using two displays. If the secondary display is positioned to the left/right or below the main display, the bottom–left and bottom–right positioning arrangements push the window down half-way off the bottom of the screen. It's hard to describe, so I made a screen video: https://cldup.com/ROSwQrInPR.mov

Hope it's easy to fix. I know Terminal is a special case.

**Comment by quinncomendant** [comment]
One other change in my environment that may be contributing to this: I recently changed to the [Input](http://input.fontbureau.com/) font. If I switch back to my old font (Luxi Mono) terminal windows arrange in the corners correctly. But other fonts do have trouble: Courier and Monaco, for example. Menlo doesn't exhibit the problem.

Actually, I discovered the Terminal preferences window itself has this problem, for all four corners of the screen. I made another screen video to show this: https://cldup.com/8Xx_D0QJII.mov
